src/markdown-pages/instant-observability/build-a-quickstart/create-alerts.mdx

```diff
@@ -0,0 +1,105 @@
+
+![add warning threshold](../../../images/instant-observability/add-warning-threshold.png)
+
+Now, specify a threshold value for warning. Say, you want the alert condition to warn you when the read response is above 0.8 for at least 5 minutes. Enter the threshold value as 0.8 and click **Save condition**. 
+
+![slow read response warning threshold](../../../images/instant-observability/warning-threshold.png)
+
+You can see your alert consition under your FlashDB alert policy now. 
+
+![flashdb slow response alert](../../../images/instant-observability/flashdb-alert-slow-response.png)
+
+Since you claimed that your database is fastest in the space, you also want to observe cache hit ratio and get a notification if it is low. You can add another alert condition under the same policy by clicking **Add a condition**.
+
+![add more alert conditions](../../../images/instant-observability/add-more-alert-conditions.png)
+
+You are familiar with the next page where you choose the category for your alert condition. Choose NRQL and click **Next, define threshold**.
+
+![add more alert conditions nrql](../../../images/instant-observability/add-more-alert-conditions-nrql.png)
+
+Enter the name _low cache hit ratio_ for your conditiona and the following NRQL query.
+
+>```bash
+> SELECT sum(fdb_cache_hits)/sum(fdb_read_responses) FROM Metric
+>```
+
+![low cache hit ratio nrql](../../../images/instant-observability/low-cache-hit-ratio-nrql.png)
+
+The next step is adding threshold values for critical violation as well as warning. The procedure is same as we did before. The only difference is that we want to get warning notification if the query returns a value _below_ 0.85 and a critical violation notification is the value falls _below_ 0.75 for at least 5 minutes.
+
+![low cache hit ratio threshold](../../../images/instant-observability/low-cache-hit-ratio-threshold.png)
+
+Save condition to add it to previoulsy defined FlashDB alert policy. Your policy will look something like the following.
+
+![alert policy conditions](../../../images/instant-observability/alert-conditions.png)
+
+### Add a notification channel
+
+To receive notifications for any opened violation, you also need to add notification channel. Move to the Notification channels tab and click **Add notification channels**.
+
+![add notification channel](../../../images/instant-observability/add-notification-channel.png)
+
+New Relic gives you multiple options such as Email, Slack, Webhook etc. to choose from. At simplest, click Email to add an email notification channel.
+
+![email notification channel](../../../images/instant-observability/email-notification-channel.png)
+
+Select the email you want to receive your notifications on, and click **Update policy**.
+
+![update policy](../../../images/instant-observability/update-policy.png)
+
```

Comment:
It would be nice to have an outro here to summarize what the user did in this lab:

## Summary